cdxev/amend/operations.py

```diff
@@ -119,53 +119,62 @@ class InferSupplier(Operation):
     """
     As we need a contact in case of a security incident, at least one of the 'author', 'supplier'
     or 'publisher' fields must be set on any component. If that's not the case, this operation
```

Comment:
As you are now just try to add further information, I would change the reasoning

Reply:
Rewrote it